File: desafio-03/anthonyfisicabsb/javascript/palindromos.js
for(var i=1; i<= 100000; i++)
{
  var eh_palidromo = true;
  var string = String(i).split("");
  
  for(var j=0; j<string.length/2; j++){
      if(string[j] != string[string.length-(1+j)]){
        eh_palidromo = false;
        break;
      }
  }
  
  if(eh_palidromo)
    console.log(i);
}

File: desafio-03/m1thrandirr/python/palindromos.py
""" Algoritmo para verificar números palindromicos em um intervalo. """

ver = 0
while ver == 0:
    intervalo = input("Digite o intervalo de inteiros a ser analisado: ").split()
    palind_lista = []
    palind = ""

    if len(intervalo) == 2:
        num_1 = int(intervalo[0])
        num_2 = int(intervalo[1])

        for i in range(num_1, num_2+1):
            palind = str(i)
            if int(palind[::-1]) == i:
                palind_lista.append(i)

        print(palind_lista)
        ver+=1

    else:
        print("Por favor, insira 2 números inteiros.")
